content/articles/building-a-simple-translation-app-using-python-for-beginners/index.md

```diff
@@ -0,0 +1,171 @@
+
+### Installing dependencies
+> An important point to note before installing PyTorch is that Google Colab comes pre-installed with PyTorch. You may need to ignore the PyTorch installation if you're using Colab and skip to the Hugging Face and Gradio installations. Only follow the steps to install PyTorch if you're installing it on your machine.
+
+Our main dependencies are going to be PyTorch, Hugging Face Transformers, and Gradio. Let's go ahead and install them.
+
+Let's first install PyTorch. This is available [here](https://pytorch.org/) on PyTorch's website. On the website, find the `Install` button and press it. On the install page, you will see a walkthrough section similar to the one shown below:
+
+![Installing PyTorch](/engineering-education/building-a-simple-translation-app-using-python-for-beginners/installing-pytorch.png)
+
+*[Image Source: PyTorch](https://pytorch.org/)*
+
+The commands for installing PyTorch vary depending on factors such as your OS, package of choice (pip, conda, source), language, and compute platform (CUDA, CPU, or ROCm). So, make sure you select your preference. It doesn't have to look like mine. Mine's shown below:
+
+![Preference](/engineering-education/building-a-simple-translation-app-using-python-for-beginners/pytorch-preference.png)
+
+*[Image Source: PyTorch](https://pytorch.org/)*
+
+To install PyTorch on my machine, I've selected the Long Term Support (LTS 1.8.1) build, Linux OS, `pip` package, Python programming language, and CPU as my compute platform of choice. If your machine has GPU support, choose accordingly.  
+
+After selecting those options, this is the command you're going to run to install PyTorch:
+
+```python
+    pip3 install torch==1.8.1+cpu torchvision==0.9.1+cpu torchaudio==0.8.1 -f https://download.pytorch.org/whl/lts/1.8/torch_lts.html
+```
+Let's now go ahead and install our other dependencies, the Hugging Face Transformers and Gradio. 
+```python
+    pip install transformers ipywidgets gradio --upgrade
+```
+> Please note that this command can be used in the command-line interface for a local machine and can also be run as a cell in the Colab notebook. You only add an exclamation `!` mark before the `pip` command when installing on the Colab. This way: `!pip install transformers ipywidgets gradio --upgrade`
+
+Our dependencies are now installed successfully!
+
+`transformers` is going to give us our translation pipeline.
+`ipywidgets` provides you with the progress bar as you're downloading the model.
+`gradio` gives you a nice way to demonstrate and interact with your ML model.
```

Comment:
The point-of-view used seems to be inconsistent. The first sentence uses "us" whereas the second and the third uses "you". Please choose one (first person or second person) and stick to that for the entire article. Please make all necessary changes in the entire draft.